pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -120,11 +120,13 @@ function LoadPackageAndExposeGlobals(package::String, mod::Module; all_globals::
     end
     new_symbols = gap_to_julia(Array{Symbol,1},new_gvar_list)
     for sym in new_symbols
-        try
-            mod.eval(:(
-                $(sym)=GAP.Globals.$(sym)
-            ))
-        catch
+        if overwrite || ! isdefined(mod,sym)
```

Comment:
So the default behaviour is changed by this PR, right? As you default `overwrite` to `false`, but the old behavior was equivalent to `overwrite = true`. So any existing code using `LoadPackageAndExposeGlobals` should be updated, right?

Reply:
Yeah, indeed. Although I would guess the code is only @mohamed-barakat s code, who specifically requested this change.

Reply:
I added tests.